includes/class-order-details.php

```diff
@@ -44,12 +46,17 @@ public function referral_args() {
 		global $affwp_od_atts;
 
 		// Set up defaults.
-		$args = apply_filters( 'affwp_odfa_referral_args', 
+		$args = apply_filters(
```

Comment:
```suggestion
		/**
		 * Filters the arguments passed to get_referrals() to populate the order details table.
		 *
		 * @since 1.0.0
		 *
		 * @param array $args         Arguments passed to get_referrals().
		 * @param int   $affiliate_id Affiliate ID.
		 */
		$args = apply_filters(
```
Add a missing DocBlock to the 'affwp_odfa_referral_args'  filter.